frontend/public/components/graphs/bar.tsx

```diff
@@ -42,42 +35,34 @@ export const BarChart: React.FC<BarChartProps> = ({
   // Max space that horizontal padding should take up. By default, 2/3 of the horizontal space is always available for the actual bar graph.
   const maxHorizontalPadding = PADDING_RATIO * width;
 
-  // SVG text element is slightly taller than font size
-  const xAxisTickLabelHeight = _.get(theme, 'independentAxis.style.tickLabels.fontSize') || _.get(theme, 'axis.style.tickLabels.fontSize', 14) * 1.25;
-  const barFootprint = barWidth + xAxisTickLabelHeight + BAR_PADDING + BAR_LABEL_PADDING;
-  const topPadding = xAxisTickLabelHeight + BAR_LABEL_PADDING; // Moving the label above the bar
-
-  // Calculate total graph height, accounting for domain padding.
-  const height = barFootprint * data.length + topPadding;
   const padding = {
-    bottom: 0,
+    bottom: 15,
     left: 0,
     right: Math.min(100, maxHorizontalPadding),
-    top: topPadding,
+    top: 0,
   };
 
-  const tickLabelComponent = <ChartLabel x={0} verticalAnchor="start" transform={`translate(0, -${xAxisTickLabelHeight + BAR_LABEL_PADDING})`} />;
-  const labelComponent = <ChartLabel x={width} />;
   return (
     <PrometheusGraph ref={containerRef} title={title} className={titleClassName} >
       {
         data.length ? (
           <PrometheusGraphLink query={query}>
-            <Chart
-              domainPadding={domainPadding}
-              height={height}
-              theme={theme}
-              width={width}
-              padding={padding}
-            >
-              <ChartAxis tickLabelComponent={tickLabelComponent} />
-              <ChartBar
-                barWidth={barWidth}
-                data={data}
-                horizontal
-                labelComponent={labelComponent}
-              />
-            </Chart>
+            {data.sort((a, b) => b.y - a.y).map((datum, index) => (
+              <>
+                <div className="graph-bar__label">{datum.x}</div>
+                <ChartBar
+                  barWidth={barWidth}
+                  data={[{x: 1, y: data.length - index, label: datum.label}]}
```

Comment:
There are a few issues I have with using the 'rank' of each consumer as the data for the chart:

1. The bar chart will always look the same, making it much less useful as it's meant to be a way to visually see relative changes at a glance.
2. There is dissonance between the bar length and the value that is displayed next to it. The bar length does not correlate to the value at all. Most users will assume that the bars represent the value, and this might lead to misinterpretation of the graph.
3. If a few of the top consumers are using significantly more resources than the rest of the consumers in the list, it will not be visually apparent (closely related to issue 2).

Reply:
+1 The intent was for each bar to shrink/grow based on its value relative to the top consumer's (always full-width) bar, making it easier to notice any outliers by visually comparing bar sizes. The bar widths should definitely not be static based on rank (1-5).

It's probably worth noting that our (WIP) 4.3 designs will remove these bars entirely. If fixing this bar behavior would require a lot of effort, maybe we could align closer with that future by dropping the bars and doing something like this for now?

![image](https://user-images.githubusercontent.com/9122899/62072166-0ed98980-b20c-11e9-9998-e8a9f0294f6e.png)

Reply:
+1 To just removing the bars. I know that fixing the overflow issue while retaining the current bar behavior (using the usage value) will not be trivial.

Reply:
Dissonance between bar length and value is also present without this change
![bar](https://user-images.githubusercontent.com/2078045/62108510-865df600-b2aa-11e9-82be-21dd9f0900f9.png) (this is namespaces page, master, without this change)

so that is not on purpose and it is a bug, Im I right ?

Reply:
@rawagner that's a bug we should fix. please open an issue. the bar length should be determined by relative usage, not in steps like this

Reply:
reported https://jira.coreos.com/browse/CONSOLE-1665

Reply:
![bar_fixed](https://user-images.githubusercontent.com/2078045/62152180-504c6080-b302-11e9-9d8e-7e329cdd0c00.png)

updated the PR:
 - reusing @TheRealJon's fix for bar length https://github.com/openshift/console/pull/2221 
 - and setting domain manually - based on highest value - results in a bar chart which has bar lenght based on values and the text does not overflow (and can be styled via simple css/added links etc)

WDYT ?

Reply:
@rawagner lgtm, didn't think about setting the domain for each bar. That should work nicely!